proto/profile/profile_service.proto

```
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package profile;
+
+option go_package = "github.com/itsLeonB/cocoon-protos/gen/go/profile";
+
+import "profile/profile_message.proto";
```

Buf (1.55.1)

import "profile/profile_message.proto": file does not exist

(COMPILE)